docs/src/main/asciidoc/logging.adoc

```diff
@@ -11,25 +11,20 @@ include::_attributes.adoc[]
 Read about the efficient use of logging API in Quarkus, configuring logging output according to your needs, and using logging adapters to unify the output from other logging APIs.
 
 Quarkus uses the JBoss Log Manager logging backend for publishing application and framework logs.
-Quarkus supports the JBoss Logging API as well as multiple other logging APIs listed in the upcoming section.
-These APIs seamlessly integrate with JBoss Log Manager.
-
-To configure your logging, you will exclusively work within your `application.properties` file.
-
-[[supported-logging-apis]]
-== Supported logging APIs
-
-Applications and components commonly utilize a logging API to log messages during runtime.
-The underlying implementation of this API is responsible for storing these messages, typically in a file.
-
-With all logs ultimately directed to the JBoss Log Manager, you have the flexibility to utilize any of the featured logging APIs available:
+Quarkus supports the JBoss Logging API as well as multiple other logging APIs, seamlessly integrated with JBoss Log Manager.
+You can use the <<logging-apis, following APIs>>:
 
 * link:https://github.com/jboss-logging/jboss-logging[JBoss Logging]
 * JDK `java.util.logging` (JUL)
 * link:https://www.slf4j.org/[SLF4J]
 * link:https://commons.apache.org/proper/commons-logging/[Apache Commons Logging]
+* link:https://logging.apache.org/log4j/2.x/[Apache Log4j 2]
+* link:https://logging.apache.org/log4j/1.2/[Apache Log4j 1]
+
+[[jboss-logging]]
+== Using JBoss Logging
 
-NOTE: By leveraging JBoss Logging within your application, you eliminate the need for additional logging dependencies.
+To use JBoss Logging, you don't need any extra dependency.
```

Comment:
This reads a little strangely. Maybe turn it around:
"No additional dependencies are needed when using the JBoss Logging API; it is automatically provided."